packages/docs/src/components/ColorSwatchList.tsx

```diff
@@ -58,14 +59,15 @@ const ColorSwatchList = ({ backgroundClass, colorNames, preface, theme }: ColorS
 
     setColorList(updatedColorList);
   }, [
+    colorList,
```

Comment:
This change is causing an endless render loop. When inspecting this, the DOM is constantly being updated.

Reply:
It seems to actually be the combination of the uniqueId change below and this line

Reply:
And it makes sense that this dependency would cause an endless loop because it's saying that if `colorList` changes to call the function, and the function always updates `colorList` at the end.